cpp/benchmarks/reduction/dictionary.cpp

```diff
@@ -32,12 +31,17 @@ void BM_reduction_dictionary(benchmark::State& state, std::unique_ptr<cudf::aggr
 {
   const cudf::size_type column_size{static_cast<cudf::size_type>(state.range(0))};
 
-  cudf::test::UniformRandomGenerator<long> rand_gen(
-    (agg->kind == cudf::aggregation::ALL ? 1 : 0), (agg->kind == cudf::aggregation::ANY ? 0 : 100));
-  auto data_it = cudf::detail::make_counting_transform_iterator(
-    0, [&rand_gen](cudf::size_type row) { return rand_gen.generate(); });
-  cudf::test::dictionary_column_wrapper<T, typename decltype(data_it)::value_type> values(
-    data_it, data_it + column_size);
+  // int column and encoded dictionary column
+  data_profile profile;
+  profile.set_null_frequency(-1);
+  profile.set_cardinality(0);
```

Comment:
Should zero be the default cardinality? Current value, 2000, seems really silly in retrospect and I see it adds a new line at almost every call site.

Reply:
Tried changing to default 0. It again causes many changes in string bench, and cuio benches. 
So, I prefer leaving as it is.